src/client/vid/glimp_sdl.c

```diff
@@ -144,12 +206,23 @@ void GLimp_GrabInput(qboolean grab);
 static void
 ShutdownGraphics(void)
 {
+	ClampDisplayIndexCvar();
+
 	if (window)
 	{
 		/* save current display as default */
 		last_display = SDL_GetWindowDisplayIndex(window);
+
+		/* or if current display isn't the desired default */
+		if (last_display != vid_displayindex->value) {
+			last_position_x = last_position_y = SDL_WINDOWPOS_UNDEFINED;
+			last_display = vid_displayindex->value;
+		}
+		else {
 		SDL_GetWindowPosition(window,
```

Comment:
please fix the indentation of this (and the next) line to respect the else {} block

Reply:
Alright, should be good.